tests/namespaced.rs

```diff
@@ -885,6 +885,76 @@ mod namespaced {
         Ok(())
     }
 
+    #[tokio::test]
+    async fn test_svc_hostname() -> anyhow::Result<()> {
+        let mut manager = setup_netns_test!(Shared);
+        let zt = manager.deploy_ztunnel(DEFAULT_NODE).await?;
+        manager
+            .service_builder("server")
+            .addresses(vec![NetworkAddress {
+                network: strng::EMPTY,
+                address: TEST_VIP.parse::<IpAddr>()?,
+            }])
+            .ports(HashMap::from([(80u16, 80u16)]))
+            .register()
+            .await?;
+        let server = manager
+            .workload_builder("server", DEFAULT_NODE)
+            .service(
+                format!("default/{SERVER_HOSTNAME}").as_str(),
+                80,
+                SERVER_PORT,
+            )
+            .register()
+            .await?;
+        let client = manager
+            .workload_builder("client", DEFAULT_NODE)
+            .uncaptured()
+            .register()
+            .await?;
+
+        run_tcp_server(server)?;
+        let srv = resolve_target(manager.resolver(), "server");
+        client
+            .run(move || async move {
+                let builder =
+                    hyper::client::conn::http2::Builder::new(ztunnel::hyper_util::TokioExecutor);
+
+                let request = hyper::Request::builder()
+                    .uri(format!("{SERVER_HOSTNAME}:{SERVER_PORT}"))
```

Comment:
This isn't quite right. The service port is `80`, but we are using `8080` here (SERVER_PORT is 8080).

We will need to perform the service port -> target port translation in inbound.rs I think

Reply:
Okay, want to make sure I am on the same page. For the test, should the connection be to the dest service IP or the dest pod IP?

> We will need to perform the service port -> target port translation in inbound.rs I think

Is this specific to when we're using a svc hostname in the authority or is the translation required when building an inbound request to a service? When the authority header is configured on the client side, would the port be the service port or the target port?

Reply:
Updated - I think the connection is using the correct IP and port, but please double check